metals/src/main/scala/scala/meta/internal/metals/UserConfiguration.scala

```diff
@@ -570,6 +587,11 @@ object UserConfiguration {
         case _ => AutoImportBuildKind.Off
       }
 
+    val scalaCliLauncher = getStringKey("scala-cli-launcher")
```

Comment:
I added this because it seems to me it should be there but maybe there was some great scheme behind not adding this that I'm not aware of.